src/client/components/hiluk-menu/hiluk-menu.tsx

```diff
@@ -65,6 +66,16 @@ export class HilukMenu extends React.Component<HilukMenuProps, HilukMenuState> {
       >{STRINGS.copySpecificUrl}</li>);
     }
 
+    if (externalView) {
+      const getUrl = new Function('cube', externalView.linkGenerator);
+      const url = getUrl(essence.getPublicEssence());
+      if (typeof url !== "string") throw new Error("link generating function must return a string");
```

Comment:
don't throw an error, console.error instead (if available). This is not a pivot bug, this is a user bug